paper/paper.md

```diff
@@ -28,16 +28,19 @@ bibliography: paper.bib
+`sweetbean` is an open-source, domain-specific declarative programming language built in Python, designed to simplify the synthesis of web-based behavioral experiments. It allows researchers to specify a behavioral experiment in declarative form as a sequence events. Once specified, `sweetbean` can compile the experiment into a `jsPsych` experiment [@de2015jspsych] for web-based behavioral study with human participants. In addition, `sweetbean` can generate prompts for conducting the same experiment with a large-language model (LLM), enabling automated alignment of LLMs with human behavior.
 
-The `SweetBean` package seamlessly integrates with packages aimed at running online behavioral experiments, such as `AutoRA` [@AUTORA-MISSING] for automated research, participant recruitment, and data collection, and `SweetPea` [@musslick2020sweetpea] for experimental design. 
+The `sweetbean` package integrates with other tools that automate aspects of behavioral research, such as `sweetpea` [@musslick2020sweetpea] for automating experimental design, or `autora` [@autora2024] for running closed-loop behavioral research studies. Together, these tools form an ecosystem for advancing behavioral research through automation.
 
 # Statement of need
 
-The design and implementation of behavioral experiments often involve complex workflows, particularly when researchers aim to run the same experiments with both human participants and large-language models—a practice that is becoming increasingly common, for example, to train models to simulate human behavior [@demircan2024evaluating] or to support AI alignment research. While platforms like `jsPsych` offer powerful solutions for running online experiments, they require knowledge of *JavaScript*, which poses a significant barrier for researchers without programming expertise. Additionally, integrating synthetic participants, such as large-language models, often necessitates ad hoc solutions, leading to inefficiencies and errors due to the need for replicating the same experiment across different codebases.
+The generation of execution of web-based experiments is a common task in the behavioral sciences. However, the process of designing and implementing such experiments can difficult, especially if researchers don't have a background in web development.  While platforms like `jsPsych` offer powerful solutions for running online experiments, they require knowledge of *JavaScript*, which poses a significant barrier for researchers without programming expertise. `sweetbean` aims to simplify this process by providing a high-level, declarative language for specifying experiments in Python. By abstracting away the details of web development, `sweetbean` allows researchers to focus on the specification of the experiment itself, rather than the implementation details.
```

Comment:
```suggestion
The generation and execution of web-based experiments is a common task in the behavioral sciences. However, the process of designing and implementing such experiments can difficult, especially if researchers don't have a background in web development.  While platforms like `jsPsych` offer powerful solutions for running online experiments, they require knowledge of *JavaScript*, which poses a significant barrier for researchers without programming expertise. `sweetbean` aims to simplify this process by providing a high-level, declarative language for specifying experiments in Python. By abstracting away the details of web development, `sweetbean` allows researchers to focus on the specification of the experiment itself, rather than the implementation details.
```